script.js

```diff
@@ -1 +1,45 @@
-'use strict';
\ No newline at end of file
+'use strict';
+
+document.addEventListener("DOMContentLoaded", function() {
+  if (document.visibilityState === 'visible') {
+    startApp();
+  } else {
+    document.addEventListener("visibilitychange", handleVisibilityChange);
+  }
+});
+
+function handleVisibilityChange() {
+  if (document.visibilityState === 'visible') {
+    startApp();
+    document.removeEventListener("visibilitychange", handleVisibilityChange);
+  }
+}
+
+function startApp() {
+  const USER_NAME = prompt("What is your name?");
```

Comment:
```suggestion
  const userName = prompt("What is your name?");
```
ти ж на момент написання коду не знаєш, яке там буде значення. Константа буде в UPPER_SNAKE_CASE коли на момент стоврення такої змінної знаєш вже, що там буде

Reply:
тобто всюди, крім випадків де я самостійно вказую значення змінною (наприклад: const EXAMPLE_VARIABLE = 23;) треба писати змінні в camelCase? І функції теж через camelCase, чи потрібно в UPPER_SNAKE_CASE?